bread/views/browse.py

```diff
@@ -192,15 +183,7 @@ def get(self, *args, **kwargs):
                 self.viewstate_sessionkey
                 and self.viewstate_sessionkey in self.request.session
             ):
-                if self.request.GET["reset"] == "2":
-                    # only reset search value
-                    session_str = self.request.session[self.viewstate_sessionkey].split(
-                        "&"
-                    )
-                    session_str = "&".join((x for x in session_str if "q=" not in x))
-                    self.request.session[self.viewstate_sessionkey] = session_str
-                else:
-                    del self.request.session[self.viewstate_sessionkey]
+                del self.request.session[self.viewstate_sessionkey]
```

Comment:
I wanted the searchquery to use the same reset-mechanism as any other filters, therefore removed some of the logic.